src/main/java/com/minecolonies/coremod/sounds/FishermanSounds.java

```diff
@@ -76,35 +69,32 @@ private Male()
     }
 
     /**
-     * Private constructor to hide the implicit public one.
-     */
-    private FishermanSounds()
-    {
-        /*
-         * Intentionally left empty.
-         */
-    }
-
-    /**
      * Plays fisherman sounds.
      *
      * @param worldIn  the world to play the sound in.
      * @param position the position to play the sound at.
      * @param isFemale the gender.
      */
-    public static void playFishermanSound(final World worldIn, final BlockPos position, final boolean isFemale)
+    @Override
+    public void playSound(final World worldIn, final BlockPos position, final boolean isFemale, final double saturation)
     {
         //Leaving it as switch-case we may add further random sound categories here (Whistling, singing, etc).
         switch (rand.nextInt(NUMBER_OF_SOUNDS + 1))
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1301)